hub/instruction_handling/call/generalities/account_rows_first_set.tex

```diff
@@ -45,10 +45,29 @@
 					{i}{\callFirstCallerAccountRowOffset}
 				\]
 			\item \If $\scenCallBalanceUpdateRequired _{i} = 1$ \Then
+				\begin{enumerate}
+				        \item \If $\locIsCall = 1$ \Then
+						\[
+							\accDecrementBalance
+							{i}{\callFirstCallerAccountRowOffset}{\locValueLo}
+						\]
+					\item \If $\locIsCallcode + \locIsDelegatecall + \locIsStaticcall = 1$ \Then
+						\[
+							\accSameBalance
+							{i}{\callFirstCallerAccountRowOffset}
+						\]
+				\end{enumerate}
+				\saNote{} \label{hub: instruction handling: call: generalities: first account rows: value transfer subtleties}
+				The above may alternatively be implemented without case disjunction as
 				\[
 					\accDecrementBalance
-					{i}{\callFirstCallerAccountRowOffset}{\locValueLo}
+					{i}{\callFirstCallerAccountRowOffset}{\locIsCall \cdot \locValueLo}
```

Comment:
What is actually implemented, see commit https://github.com/Consensys/linea-constraints/pull/512/commits/2087e3f94e4a4522334e0d05897def017d210a9d